src/main/resources/application.yml

```diff
@@ -31,7 +31,7 @@ cron:
 
 workers:
   askForReplicatePeriod: ${IEXEC_ASK_REPLICATE_PERIOD:5000}
-  requiredWorkerVersion: ${IEXEC_CORE_REQUIRED_WORKER_VERSION:6.1.0} #leave empty will allow any worker version
+  requiredWorkerVersion: ${IEXEC_CORE_REQUIRED_WORKER_VERSION:} #leave empty will allow any worker version
```

Comment:
Do we prefer leaving it empty?

Reply:
I think it's simpler for testing. We can fix it later when releasing the major version. I think we can also change it to `requiredMinimumWorkerVersion`.

Reply:
I agree with the simplification concerning the current usage.
About requiredMinimumWorkerVersion, we should introduce this kind of feature yes :+1: